src/mscorlib/src/System/Collections/Generic/Comparer.cs

```diff
@@ -199,9 +199,6 @@ public Int32EnumComparer()
             Debug.Assert(typeof(T).IsEnum, "This type is only intended to be used to compare enums!");
         }
         
-        // Used by the serialization engine.
-        private Int32EnumComparer(SerializationInfo info, StreamingContext context) { }
```

Comment:
These deserialization ctors should not be removed.  We have tests for this in CoreFx (I know because I recently added them)... they pass with this change?